Source/Atomic/Atomic2D/ParticleEmitter2D.cpp

```diff
@@ -312,8 +316,7 @@ void ParticleEmitter2D::Update(float timeStep)
             emitParticleTime_ -= timeBetweenParticles;
         }
 
-        if (emissionTime_ > 0.0f)
-            emissionTime_ = Max(0.0f, emissionTime_ - timeStep);
+        emissionTime_ = Max(0.0f, emissionTime_ - timeStep);

```

Comment:
Sorry, I'm still torn about the removal of this line, since there are no guards, and the next rebase or merge from Urho3D can easily bring it back.